### ISSUE TITLE
i have something wrong when i train the model

### ISSUE THREAD

**Issue by lionhew** [OpenNMT/OpenNMT-py#1410]
![0V20LX)`)$ T4`584I0MMS3](https://user-images.githubusercontent.com/35457016/56788927-3c8cec00-6833-11e9-8d6c-98c0d96e9eef.png)

Is it my dataset or where there is a problem that caused this problem, and if so, how should I correct it, thank you.

torch 1.0

> I used the following command：
> python preprocess.py -train_src data/giga/train.article.txt -train_tgt data/giga/train.title.txt -valid_src data/giga/valid.article.txt -valid_tgt data/giga/valid.title.txt -save_data data/giga/GIGA -src_seq_length 10000 -dynamic_dict -share_vocab -shard_size 100000
> 
> python train.py -data data/giga/GIGA -save_model models/giga -optim adagrad -copy_attn -reuse_copy_attn -batch_size 4 -valid_batch_size 4 -learning_rate 0.15 -gpu_ranks 0 -dropout 0. -train_steps 200000 

I am a beginner if I have something wrong, I hope to get your guidance.

**Comment by vince62s** [comment]
If you are a beginner please follow the tutos in the forum or on the FAQ page first, and check if all of this work.
obviously something is wrong in your setup since I see 100000 examples in your valid set, way too much.
